src/Evolution/Particles/MonteCarlo/CellVolume.hpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "Utilities/Gsl.hpp"
+
```

Comment:
just include DataStructures/Tensor/TypeAliases.hpp and give forward declarations (wrapped in \cond and \endcond) for DataVector, Mesh, and gsl::not_null

You'll need to move the includes to the cpp file